data-prepper-plugins/kafka-source/README.md

```diff
@@ -0,0 +1,131 @@
+# Kafka source
+
+This is the Data Prepper Kafka source plugin that reads records from Kafka topic. It uses the consumer API provided by Kafka to read messages from the broker.
+
+The Kafka source plugin supports OpenSearch 2.0.0 and greater.
+
+## Usages
+
+The Kafka source should be configured as part of Data Prepper pipeline yaml file.
+
+## Configuration Options
+
+```
+log-pipeline:
+  source:
+    kafka:
+      bootstrap_servers:
+        - 127.0.0.1:9092,localhost:9093, 127.0.0.1:9094
```

Comment:
This doesn't seem to match the actual YAML configured or desired.

It should be:

```
bootstrap_servers:
  - 127.0.0.1:9092
  - localhost:9093
  - 127.0.0.1:9049
```

This is equivalent to:

```
bootstrap_servers: [ "127.0.0.1:9092", "localhost:9039", "127.0.0.1:9094" ]
```


In code, this will be represented as:

```
private List<String> bootstrapServers;
```